cluster/plan.go

```diff
@@ -49,8 +49,9 @@ const (
 	KubeletDockerConfigPath    = "/var/lib/kubelet/config.json"
 
 	// MaxEtcdOldEnvVersion The versions are maxed out for minor versions because -rancher1 suffix will cause semver to think its older, example: v1.15.0 > v1.15.0-rancher1
-	MaxEtcdOldEnvVersion = "v3.2.99"
-	MaxK8s115Version     = "v1.15"
+	MaxEtcdOldEnvVersion   = "v3.2.99"
+	MaxK8s115Version       = "v1.15"
+	MaxEtcdPort4001Version = "v3.4.3-rancher99"
```

Comment:
should this be 3.4.13?

Reply:
This is to handle the `-rancher` suffix which leads to `v3.14.3-rancher1` being marked as lower than `v3.14.3`